src/AST-Core/RBLiteralValueNode.class.st

```diff
@@ -40,8 +40,9 @@ RBLiteralValueNode class >> value: aValue start: aPosition stop: anotherPosition
 { #category : #comparing }
 RBLiteralValueNode >> = anObject [ 
 	^ super = anObject 
-		and: [ self value = anObject value 
-		and: [ self value species = anObject value species ] ]
+		and: [ self value = anObject value
```

Comment:
I think #literalEqual: should be used here:
```Smalltalk
and: [ self value literalEqual: anObject value ]
```
And other conditions can be removed this way.

Also it would be nice to have a test describing this change.